src/screens/monitor/components/monitorDoubleCard/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box } from '@mui/system';
+import React from 'react';
+import { Card as MuiCard, CardContent, Grid, Typography } from '@mui/material';
+
+export interface MonitorCardProps {
+  titleLeft: string | number | null;
+  titleRight: string | number | null;
+  subtitleLeft: string | number | null;
+  subtitleRight: string | number | null;
+}
+
+export default function MonitorDoubleCard(props: MonitorCardProps) {
+  const { titleLeft, titleRight, subtitleLeft, subtitleRight } = props;
+
+  return (
+    <MuiCard sx={{ minWidth: 275, minHeight: 120 }}>
+      <CardContent>
+        <Grid container spacing={8}>
+          <Grid item xs={5}>
+            <Box
+              sx={{
+                display: 'flex',
+                justifyContent: 'center',
+                alignItems: 'center',
+                flexDirection: 'column',
+              }}
```

Comment:
We should create a reusable styling for this two boxes that are having the same style.